### PR TITLE
Redis - fixes

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis-cache.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-14/docs-ref-services/redis-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180124173553-114%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/redis-cache.md [hunk 1]
```diff
@@ -23,23 +23,28 @@ Redis is an advanced key-value store, where keys can contain data structures suc
 
 Learn more about [Azure Redis Cache](https://docs.microsoft.com/azure/redis-cache/).
 
-## Management Package
+## Management package
 
-#### Install the packages with npm
+#### Install the npm module
```
Comment (clarkio) [inline docs-ref-services/redis-cache.md]: Remove extra `#` and `the` for consistency with the template

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis-cache.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-14/docs-ref-services/redis-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180348437588-114%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis-cache.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-14/docs-ref-services/redis-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180349495918-114%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis-cache.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-14/docs-ref-services/redis-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180416259343-114%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/redis-cache.md [hunk 2]
```diff
@@ -23,32 +23,36 @@ Redis is an advanced key-value store, where keys can contain data structures suc
...
 ```javascript
 const msRestAzure = require('ms-rest-azure');
 const AzureMgmtRedisCache = require('azure-arm-rediscache');
 
+const subscriptionId = 'your-subscription-id';
+
 msRestAzure.interactiveLogin().then(credentials => {
-  const client = new AzureMgmtRedisCache(credentials, 'my-subscription-id');
+  const client = new AzureMgmtRedisCache(credentials, subscriptionId);
   client.redis.listByResourceGroup('testResourceGroup').then(result => {
```
Comment (jawache) [inline docs-ref-services/redis-cache.md]: Inconsistent with subscriptionId above, lets break this out into `const resourceGroupName = 'your-resource-group';`

Comment (jawache) [inline docs-ref-services/redis-cache.md]: Highlighted the wrong line, meant 48 :)

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis-cache.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-14/docs-ref-services/redis-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707181351176673-114%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**